pkg/daemon/writer.go

```diff
@@ -102,10 +104,14 @@ func (nw *clusterNodeWriter) SetWorking(client corev1client.NodeInterface, liste
 // SetUnreconcilable sets the state to Unreconcilable.
 func (nw *clusterNodeWriter) SetUnreconcilable(err error, client corev1client.NodeInterface, lister corev1lister.NodeLister, node string) error {
 	glog.Errorf("Marking Unreconcilable due to: %v", err)
+	// truncatedErr caps error message at a reasonable length to limit the risk of hitting the total
+	// annotation size limit (256 kb) at any point
+	truncatedErr := fmt.Sprintf("%.2000s", err.Error())
 	annos := map[string]string{
 		constants.MachineConfigDaemonStateAnnotationKey:  constants.MachineConfigDaemonStateUnreconcilable,
-		constants.MachineConfigDaemonReasonAnnotationKey: err.Error(),
+		constants.MachineConfigDaemonReasonAnnotationKey: truncatedErr,
 	}
+	MCDState.WithLabelValues(constants.MachineConfigDaemonStateUnreconcilable, truncatedErr).SetToCurrentTime()
```

Comment:
This function was introduced in 4.3 release. It seems to me that this is related to telemetry which we added in 4.3, perhaps we can skip these.

Reply:
yea we need to drop these calls

Reply:
dropped. thanks for catching.